docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -500,12 +504,22 @@ and have at least the given labels.
 The flags can also appear anywhere later on the command line.
 
 The acceptable flags include all those of `kubectl` except for
-`--context`.  This command also accepts the following flags.
+`--context`.  BUT do not actually supply any of these flags; this
+command handles them incorrectly.
```

Comment:
I wouldn't put the reason here, it's either supported or not. If it's  not maybe: "currently --context is not supported" ?

Reply:
I would not worry about word-smithing this too carefully, it is being removed in #1444 .